### PR TITLE
Add list-view, tile-view and sort-type icons

### PR DESCRIPTION
**Pull request by mdedys**

### CODE REVIEW

**Comment by mdedys** [comment]
Nick Dingle provided these icons and told me they have been approved a tier1 icons. I figured I would add them here.

**Comment by dlockhart** [comment]
@mdedys looks like you've got some conflicts. Also be aware that there are 2 main branches now:
- `master` is a Polymer hybrid element and is the `4.x` version
- the `3.x` branch is the Polymer 1 element and is what's currently used in Brightspace

So if you need these icons in Brightspace or in a FRA that hasn't upgraded to the hybrid web components yet, you'll need to also put this change into the `3.x` branch.

**Comment by mdedys** [comment]
Fixing for conflicts on the way. FRA using this uses the 4.x version of this library. I should be good to just put this into master correct?

File: images/tier1/sort-type.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 18 18">
```
Comment (dbatiste) [inline images/tier1/sort-type.svg]: Should this one be flipped for RTL?

Comment (dlockhart) [inline images/tier1/sort-type.svg]: Dave's right -- I think this on should be flipped in RTL by adding a `mirror-in-rtl="true"` attribute to the SVG. See this for more info:
https://github.com/BrightspaceUI/icons#bidirectionality

**Comment by mdedys** [comment]
I don't have write access so I can't merge this PR.

**Comment by dlockhart** [comment]
Merged and released as version `v4.3.0`.

**Pull request merged**